sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,31 +1,25 @@
-# Release History
-
-## 4.0.0-beta.5 (Unreleased)
-
-### Features Added
-
-- Add support for Backup/Restore operations for Key Vault HSM.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
-## 4.0.0-beta.4 (2024-01-16)
-
-### Bugs Fixed
-
-- [[#4754]](https://github.com/Azure/azure-sdk-for-cpp/issues/4754) Thread safety for authentication policy.
-
-## 4.0.0-beta.3 (2023-04-11)
-
-### Features Added
-
-- Added support for challenge-based and multi-tenant authentication.
-
-## 4.0.0-beta.2 (2023-03-07)
-
-### Features Added
-
-- Initial preview of Settings API.
+# Release History
+
+## 4.0.0-beta.5 (2024-08-06)
+
+### Features Added
+
+- Add support for Backup/Restore operations for Key Vault HSM.
+
+## 4.0.0-beta.4 (2024-01-16)
+
+### Bugs Fixed
+
+- [[#4754]](https://github.com/Azure/azure-sdk-for-cpp/issues/4754) Thread safety for authentication policy.
+
+## 4.0.0-beta.3 (2023-04-11)
+
+### Features Added
+
+- Added support for challenge-based and multi-tenant authentication.
+
+## 4.0.0-beta.2 (2023-03-07)

```

Comment:
We started this off with beta.2?

Looks like this is a typo (maybe copy-paste). This is the keyvault administration changelog. 

> - Initial preview of **Settings** API.